decentraland/unity-explorer
# Fix: Nametag is not dynamic with wearable bounds (#1683)

## What does this PR change?

It changes how the height of the nametags is calculated. Nametags will always be above the avatar, no matter what wearables are equipped.

## How to test the changes?

1. Launch the explorer
2. Change the wearables of the avatar, use different hats for example.
3. Meet other players and see how your nametags have different heights.
<img width="343" alt="Screenshot 2024-08-30 135212" src="https://github.com/user-attachments/assets/1d288dfa-3839-4669-b6d7-6c07cb57479b">

The name tag should move up or down depending on the height of the avatar.
If you test in Unity editor, you can uncomment the avatar bounds visualization code and see something like this:
<img width="626" alt="Screenshot 2024-08-30 135445" src="https://github.com/user-attachments/assets/abeca046-5fec-46f2-95b8-6eaf19f9931f">
